src/Workspaces/CSharp/Portable/Utilities/CSharpTypeStyleHelper.State.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Utilities
 {
     internal partial class CSharpTypeStyleHelper
     {
-        protected class State
+        protected struct State
         {
             private readonly Dictionary<UseVarPreference, ReportDiagnostic> _styleToSeverityMap;
```

Comment:
📝 I believe this could be converted to three fields

Reply:
agreed.  will fix this up.

Reply:
inlined practically all of tihs.